201-vm-domain-join/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
if the intent here is to create an ADDC then use the module in the repo for this (instead of add another one) - see: https://github.com/Azure/azure-quickstart-templates/tree/master/101-anf-smb-volume/prereqs